fineract-loan/src/main/java/org/apache/fineract/accounting/producttoaccountmapping/service/SavingsProductToGLAccountMappingHelper.java

```diff
@@ -229,6 +249,61 @@ public void handleChangesToSavingsProductToGLAccountMappings(final Long savingsP
                         savingsProductId, CashAccountsForSavings.ESCHEAT_LIABILITY.getValue(), changes);
             break;
             case ACCRUAL_PERIODIC:
+                // asset
+                mergeSavingsToAssetAccountMappingChanges(element, SavingProductAccountingParams.SAVINGS_REFERENCE.getValue(),
+                        savingsProductId, AccrualAccountsForSavings.SAVINGS_REFERENCE.getValue(),
```

Comment:
Same here

Reply:
Done! similar as above, new base method reused

Reply:
Are you sure you have pushed the changes? i dont see any changes here